MobileCenter/MobileCenterTests/Util/MSMockUserDefaults.m

```diff
@@ -0,0 +1,68 @@
+#import <Foundation/Foundation.h>
+#import <OCMock/OCMock.h>
+#import "MSMockUserDefaults.h"
+
+@interface MSMockUserDefaults()
+
+@property(nonatomic) NSMutableDictionary<NSString*,NSObject*> *dictionary;
+@property(nonatomic) id mockUserDefaults;
+
+@end
+
+@implementation MSMockUserDefaults
+
+@synthesize dictionary;
+@synthesize mockUserDefaults;
+
+- (instancetype)init {
```

Comment:
It terms of code convention we have this rule:  Empty lines not preceded by a comment within a code block is not allowed.

For instance this:
```
       id object;
       [invocation getArgument:&object atIndex:2];
 
       if (!object) {
         return;
       }
```

Should be this:
```
       id object;
       [invocation getArgument:&object atIndex:2];

       // Don't store nil objects.
       if (!object) {
         return;
       }
```